crates/util/auto-https/src/lib.rs

```diff
@@ -45,7 +45,6 @@ pub fn axum_acceptor(
 
     // Define our server configuration, using the ACME certificate resolver.
     let mut rustls_config = ServerConfig::builder()
-        .with_safe_defaults()
```

Comment:
Apparently the ServerConfig is now safe by default. I need to dig it up, this was mentioned either in the hyper issue tracker or in their changelog.